pkgs/tools/misc/rust-hello-world/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchgit }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rustHelloWorld";
+  version = "0.1.0";
+
+  src = fetchgit {
```

Comment:
`fetchFromGitea` is preferred over just a regular `fetchGit`. git forge specific fetchers will fetch tarballs, which is faster and less strain on infra, both for codeberg and nix.